src/Microsoft.AspNet.Mvc.Xml/DefaultWrapperProviderFactoryProvider.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNet.Mvc.Xml
+{
+    public class DefaultWrapperProviderFactoryProvider : IWrapperProviderFactoryProvider
```

Comment:
Also this is not a provider, its just the collection of the wrapper provider factories.

So it can be as simple as an IList<IWrapperProviderFactory> and we have a single static method providing the default values